[2024-05-08T19:52:30.940+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-08T19:52:30.977+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 7.3_filesensor.waiting_file scheduled__2022-07-27T00:00:00+00:00 [queued]>
[2024-05-08T19:52:30.984+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 7.3_filesensor.waiting_file scheduled__2022-07-27T00:00:00+00:00 [queued]>
[2024-05-08T19:52:30.985+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-05-08T19:52:30.999+0000] {taskinstance.py:2330} INFO - Executing <Task(FileSensor): waiting_file> on 2022-07-27 00:00:00+00:00
[2024-05-08T19:52:31.004+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2240) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-08T19:52:31.005+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', '7.3_filesensor', 'waiting_file', 'scheduled__2022-07-27T00:00:00+00:00', '--job-id', '724', '--raw', '--subdir', 'DAGS_FOLDER/07_FileSensor.py', '--cfg-path', '/tmp/tmplbgddfen']
[2024-05-08T19:52:31.005+0000] {standard_task_runner.py:63} INFO - Started process 2241 to run task
[2024-05-08T19:52:31.006+0000] {standard_task_runner.py:91} INFO - Job 724: Subtask waiting_file
[2024-05-08T19:52:31.053+0000] {task_command.py:426} INFO - Running <TaskInstance: 7.3_filesensor.waiting_file scheduled__2022-07-27T00:00:00+00:00 [running]> on host ccc314eb571d
[2024-05-08T19:52:31.126+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='7.3_filesensor' AIRFLOW_CTX_TASK_ID='waiting_file' AIRFLOW_CTX_EXECUTION_DATE='2022-07-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-07-27T00:00:00+00:00'
[2024-05-08T19:52:31.127+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-08T19:52:31.139+0000] {baseoperator.py:399} WARNING - FileSensor.execute cannot be called outside TaskInstance!
[2024-05-08T19:52:31.150+0000] {base.py:84} INFO - Using connection ID 'fs_default' for task execution.
[2024-05-08T19:52:31.151+0000] {filesystem.py:85} INFO - Poking for file /tmp/file.txt
[2024-05-08T19:52:31.152+0000] {filesystem.py:89} INFO - Found File /tmp/file.txt last modified: 20240508195159
[2024-05-08T19:52:31.153+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-05-08T19:52:31.155+0000] {filesystem.py:85} INFO - Poking for file /tmp/file.txt
[2024-05-08T19:52:31.155+0000] {filesystem.py:89} INFO - Found File /tmp/file.txt last modified: 20240508195159
[2024-05-08T19:52:31.156+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-08T19:52:31.163+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=7.3_filesensor, task_id=waiting_file, run_id=scheduled__2022-07-27T00:00:00+00:00, execution_date=20220727T000000, start_date=20240508T195230, end_date=20240508T195231
[2024-05-08T19:52:31.179+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-08T19:52:31.199+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-08T19:52:31.201+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
